common/lib/capa/capa/templates/codeinput.html

```diff
@@ -1,12 +1,16 @@
+<%page expression_filter="h"/>
```

Comment:
@robrap Can you please take a quick look at this file from xss perspective. Thanks.